packages/sidecar/src/formatters/human/utils.ts

```diff
@@ -153,23 +153,28 @@ export function colorizeTime(time: string): string {
  * Colorize source based on envelope source
  */
 export function colorizeSource(source: string): string {
+  const uppercaseSource = source.toUpperCase();
```

Comment:
Not sure if I actually like the upper case source. It looks  especially next to fully upper case level/type.

Maybe pull the type before the source and keep the lower case but bracketed?